xobjects/context.py

```diff
@@ -70,13 +72,13 @@ def sort_classes(classes):
         for cl in cllist:
             if not cl.__name__ in cdict:
                 cdict[cl.__name__] = cl
-                lst.append(cl)
+                classes.append(cl)
```

Comment:
Maybe a comment on the fact that we are appending to the list we are loping on is a good idea.